core/src/main/java/com/ritense/valtimo/service/GlobalSettingsService.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.valtimo.service;
+
+import com.ritense.valtimo.domain.global.GlobalSettings;
+import com.ritense.valtimo.repository.GlobalSettingsRepository;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+public class GlobalSettingsService {
+
+    private final GlobalSettingsRepository globalSettingsRepository;
+
+    public GlobalSettingsService(GlobalSettingsRepository globalSettingsRepository){
+        this.globalSettingsRepository = globalSettingsRepository;
+    }
+
+    public Optional<GlobalSettings> getGlobalSettings() {
+        return globalSettingsRepository.findAll().stream().findFirst();
```

Comment:
This table will always have a single row. Thats a first.

Do @theo-ritense @ivo-ritense have an opinion on this?